### PR TITLE
Add Google Pay configurations to Ready To Pay

### PR DESCRIPTION
**Pull request by Weetbix**
This PR adds the google pay configuration details to the ready to pay responses.

- Very new to our API and open api, so looking for suggestions around folder structure, naming etc. 
- I did not create a common file for the Google Pay merchant configuration to remove this duplication [here](https://github.com/Rebilly/api-definitions/blob/master/openapi/components/schemas/PaymentCards/DigitalWallets.yaml#L26). If it was common, we would have to delete the "Leave blank if using FramePay." description, as it wouldn't make any sense.
- I am not sure if this is the correct way to form this kind of a structure, as the `feature` type becomes: `(Apple Pay Digital Wallet (object or null)) or (Google Pay Digital Wallet (object or null))` instead of something more sensible like `Apple Pay Digital Wallet or Google Pay Digital Wallet or null`. I tried a few different configurations and didn't find anything better, and couldn't find a similar setup in our API.

### CODE REVIEW

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/ApplePayFeature.yaml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 type: object
-title: Digital Wallet
+title: Apple Pay Digital Wallet
 properties:
-  name:
-    type: string
+  method:
```
Comment (subenson) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/ApplePayFeature.yaml:4]:
```suggestion
  name:
```

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+type: object
+title: Google Pay Digital Wallet
+properties:
+  method:
```
Comment (subenson) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml:4]:
```suggestion
  name:
```

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    enum:
+      - Google Pay
+  merchantName:
+    description: The merchant name in Google Pay.
+    type: string
+    example: "Test Merchant"
+  merchantOrigin:
+    description: The merchant origin in Google Pay. The fully qualified domain name.
+    type: string
+    example: "www.example.com"
```
Comment (slavcoclone) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml:16]: Please add newlines at the end of all files per code-style

https://github.com/Rebilly/api-definitions/blob/main/.editorconfig#L8

Comment (Weetbix) [reply openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml]: Ah thanks, I needed to install the editor config plugin for vscode 🤦

**Review by agualis — APPROVED** [review]

**Review by XaosSintez — APPROVED** [review]

**Review by akozhevnikov-rebilly — APPROVED** [review]

**Pull request merged**